lms/templates/registration/password_reset_confirm.html

```diff
@@ -91,7 +91,11 @@ <h3 class="message-title">{% trans "We're sorry, our systems seem to be having t
         </div>
 
         <p class="instructions">
-        {% trans 'Please enter your new password twice so we can verify you typed it in correctly. <br /> Required fields are noted by <strong class="indicator">bold text and an asterisk (*)</strong>.' %}
+        {% trans 'Please enter your new password twice so we can verify you typed it in correctly.' %}
+        <br />
+        {% blocktrans with start_em='<strong class="indicator">' end_em='</strong>' %}
```

Comment:
I would almost say to call this "start_bold" or "bold_start" or "link_start", just to be a bit clearer to translators. If they don't know how to handle html they probably won't know what "em" means.